ios/RNSScreenContainer.m

```diff
@@ -102,33 +102,18 @@ - (void)updateContainer
   }
 
   // detect if new screen is going to be activated
-  BOOL activeScreenAdded = NO;
+  RNSScreenView* activeScreenAdded = NULL;
   for (RNSScreenView *screen in _reactSubviews) {
     if (screen.active && ![_activeScreens containsObject:screen]) {
-      activeScreenAdded = YES;
+      activeScreenAdded = screen;
     }
   }
-
-  // if we are adding new active screen, we perform remounting of all already marked as active
-  // this is done to mimick the effect UINavigationController has when willMoveToWindow:nil is
-  // triggered before the animation starts
+  
   if (activeScreenAdded) {
-    for (RNSScreenView *screen in _reactSubviews) {
-      if (screen.active && [_activeScreens containsObject:screen]) {
-        [self detachScreen:screen];
-        // disable interactions for the duration of transition
-        screen.userInteractionEnabled = NO;
-      }
-    }
-
-    // add new screens in order they are placed in subviews array
-    for (RNSScreenView *screen in _reactSubviews) {
-      if (screen.active) {
-        [self attachScreen:screen];
-      }
-    }
+    activeScreenAdded.userInteractionEnabled = NO;
```

Comment:
This code has been here to prevent view we are sliding away from responding to touches. If you delete this you will be able to do double taps on buttons etc. which isn't desirable. Also if you delete code that trigger `willMoveToWindow` this will likely interfere with the way keyboard handling work. What needs to be tested is the case when you have a keyboard open and you push new screen and also the opposite case when you pop from screen with keyboard opened.